src/nwm_routing/src/nwm_routing/__main__.py

```diff
@@ -431,7 +431,7 @@ def main_v02(argv):
         else:
             # TODO: Consider adding option to read cold state from route-link file
             waterbodies_initial_ds_flow_const = 0.0
```

Comment:
@jameshalgren Do we need to put this same "if restart_parameters.get" check in main_v03?

Reply:
We do. I think that would go into preprocess.py.

Reply:
Updated in preprocess.py